report/src/main/webapp/app/services/metadata.service.js

```diff
@@ -287,6 +291,14 @@ define(['angularAMD', 'metadataCacheService', 'metadataEndpointService'],
           }
         }
 
+        function conditionallyPassed() {
+          decoratedObject.total++;
+          decoratedObject.conditionallyPassed++;
```

Comment:
Do we need to increase `passed` value here or not?

Reply:
We can increase `passed`. Now, it is evaluating there: https://github.com/Cognifide/aet/blob/f2dbc057f4278e8335583c8cfad43a43b0deb0ee/report/src/main/webapp/app/layout/sidepanel/sidepanel.view.html#L119
What do you think, which way will be better?
I am not 100% sure, why it works in toolbar:
https://github.com/Cognifide/aet/blob/f2dbc057f4278e8335583c8cfad43a43b0deb0ee/report/src/main/webapp/app/layout/toolbar/toolbarTop.view.html#L56
but as you can see on the screen shot, it works correctly:
https://github.com/Cognifide/aet/blob/f2dbc057f4278e8335583c8cfad43a43b0deb0ee/documentation/src/main/wiki/assets/suiteReport/layout-conditionally-passed.PNG
I think @m-suchorski know it better ;)